packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/bitmap.dart

```diff
@@ -132,7 +120,7 @@ class BitmapDescriptor {
   // This is needed in Web to re-hydrate BitmapDescriptors that have been
   // transformed to JSON for transport.
   // TODO(https://github.com/flutter/flutter/issues/70330): Clean this up.
-  BitmapDescriptor.fromJson(dynamic json) : _json = json {
+  BitmapDescriptor.fromJson(Object json) : _json = json {
```

Comment:
I asked in Dart channel, and they said `Object`/`Object?` is generally preferred these days over dynamic.

I did a pass over the whole plugin, and have now removed ~80% of the usage of `dynamic` in favor of `Object`/`Object?`. Exceptions:
- Dealing with json of an unknown source; that would involve more careful rewriting than I wanted to mix into this PR.
- Method channel interactions, since I think "who knows" (aka `dynamic`) is a good expression of the type there, unfortunately, without doing a cross-implementation examination of everything and what can and can't be null.
- A couple of interfaces where things get passed through to places that already take dynamic. Maybe not the best choice, but seemed safest for now.

It definitely tightened up a bunch of nullability checks, so it's something we should look for in other plugins when we do second-pass nullability evaluations .